src/main/resources/init_db.sql

```diff
@@ -1,12 +1,45 @@
 CREATE SCHEMA `taxi_service` DEFAULT CHARACTER SET utf8;
 
+CREATE TABLE `cars`
+(
+    `id`              bigint       NOT NULL AUTO_INCREMENT,
+    `car_name`        varchar(100) NOT NULL,
+    `manufacturer_id` bigint  DEFAULT NULL,
+    `deleted`         tinyint DEFAULT NULL,
+    PRIMARY KEY (`id`),
+    KEY `manufacturer_id` (`manufacturer_id`),
+    CONSTRAINT `cars_ibfk_1` FOREIGN KEY (`manufacturer_id`) REFERENCES `manufacturers` (`id`)
+) ENGINE = InnoDB
+  DEFAULT CHARSET = utf8;
+
+CREATE TABLE `drivers`
+(
+    `id`             bigint       NOT NULL AUTO_INCREMENT,
+    `driver_name`    varchar(100) NOT NULL,
```

Comment:
```suggestion
    `name`    varchar(100) NOT NULL,
```
it's better not to have table name duplicated in column names, same for all columns
if possible, fix this

Reply:
Done